pac4j-oauth/src/main/java/org/pac4j/oauth/profile/vk/VkProfileDefinition.java

```diff
@@ -60,7 +61,7 @@ public VkProfileDefinition() {
             CAN_WRITE_PRIVATE_MESSAGE})
             .forEach(a -> primary(a, Converters.BOOLEAN));
         primary(BIRTH_DATE, new DateConverter("dd.MM.yyyy"));
-        primary(SEX, Converters.GENDER);
+        primary(SEX, new GenderConverter("2", "1"));
```

Comment:
And keep the current `Converters.GENDER`?

Reply:
Because f or female undoubtedly represents Gender.FEMALE, m or male undoubtedly represents Gender.MALE
But 1 and 2 may be different in different situations. For example, wechat uses 1 for male and 2 for female.
https://open.weixin.qq.com/cgi-bin/showdocument?action=dir_list&t=resource/res_list&verify=1&id=open1419316518&token=&lang=zh_CN
![image](https://user-images.githubusercontent.com/5005199/42298857-b6502ec6-803a-11e8-8b95-c0c77df83bd1.png)
![image](https://user-images.githubusercontent.com/5005199/42298864-bf570328-803a-11e8-8074-488d41defb69.png)

If we hardcode 1 and 2, we can't define wechat.